packages/fxa-settings/src/components/NotificationPromoBanner/en.ftl

```diff
@@ -0,0 +1,4 @@
+## Notification Promo Banner component
+account-recovery-notification-cta = Create
```

Comment:
There are two notifications in the one of the screenshots, both have a button with `Create`. Noting here that this string seems to be only for the Account Recovery Key, which is good. If you will be implementing the second notification (`Secure your account...`) then please be sure to create a second string for it's associated button.

Reply:
Thanks for the callout, the screenshot with two was just as a concept. We would add new strings for any additional things.